isis/src/base/objs/ProcessMapMosaic/ProcessMapMosaic.truth

```diff
@@ -546,26 +562,27 @@ End_Object
 
 Object = History
   Name      = IsisCube
+  StartByte = 71980
+  Bytes     = 3020
 End_Object
 
 Object = OriginalLabel
   Name      = IsisCube
+  StartByte = 66631
+  Bytes     = 5349
 End_Object
 End
 
 Testing Mosaic containing cropped image.
-unittest: Initializing mosaic
-0% Processed
10% Processed
20% Processed
30% Processed
40% Processed
50% Processed
60% Processed
70% Processed
80% Processed
90% Processed
100% Processed
-unittest: Mosaicking unitTest_crop.cub
-0% Processed
10% Processed
20% Processed
30% Processed
40% Processed
50% Processed
60% Processed
70% Processed
80% Processed
90% Processed
100% Processed
 /usgs/cpkgs/isis3/data/base/testData/ProcessMapMosaic/unitTest_crop.cub is inside the mosaic
-unittest: Mosaicking unitTest_nocrop.cub
-0% Processed
10% Processed
20% Processed
30% Processed
40% Processed
50% Processed
60% Processed
70% Processed
80% Processed
90% Processed
100% Processed
 /usgs/cpkgs/isis3/data/base/testData/ProcessMapMosaic/unitTest_nocrop.cub is inside the mosaic
 Mosaic label: 
 Object = IsisCube
   Object = Core
+    StartByte   = 65537
```

Comment:
is the unitTest.exclude file not being used (https://github.com/USGS-Astrogeology/ISIS3/blob/cmake/isis/src/base/objs/ProcessMapMosaic/unitTest.exclude)?

the file has the following keys that it should exclude:
```
ByteOrder
StartByte
Bytes
TileSamples
TileLines
```